sdc/functions/numpy_like.py

```diff
@@ -393,3 +395,106 @@ def nanprod_impl(a):
         return c
 
     return nanprod_impl
+
+
+def get_pool_size():
+    if sdc.config.config_use_parallel_overloads:
+        return numba.config.NUMBA_NUM_THREADS
+    else:
+        return 1
+
+@sdc_overload(get_pool_size)
+def get_pool_size_overload():
+    pool_size = get_pool_size()
+    def get_pool_size_impl():
+        return pool_size
+
+    return get_pool_size_impl
+
+def get_chunks(size, pool_size=0):
+    if pool_size == 0:
+        pool_size = get_pool_size()
+
+    chunk_size = size//pool_size + 1
```

Comment:
```suggestion
    chunk_size = (size - 1)//pool_size + 1
```

this is the correct formula